### DISCUSSION TITLE
Add an Angular Quickstart

### DISCUSSION THREAD

**Discussion by Isaiah-Hamilton** [supabase/supabase#16851] [Contribute to Supabase]
# Improve documentation

## Describe the problem

There is no Angular Quickstart.

## Describe the improvement

add an Angular Quickstart just like the [React Quickstart](https://supabase.com/docs/guides/getting-started/quickstarts/reactjs)

**Comment by Biki-das** [comment]
@Isaiah-Hamilton can i work on this?

**Comment by KIET7UKE** [comment]
@Isaiah-Hamilton Hey, I'm interested in working on this.

**Comment by akshatmishra25** [comment]
@Isaiah-Hamilton I would love to work on this could you kindly assign this task to me

**Comment by Isaiah-Hamilton** [comment]
Sorry for the late delay @Biki-das @KIET7UKE and @akshatmishra25. it looks like @jobayer12 has already submitted a pull request. I will let you know if there are any angular issue that need to get done.

**Comment by KIET7UKE** [comment]
> Sorry for the late delay @Biki-das @KIET7UKE and @akshatmishra25. it looks like @jobayer12 has already submitted a pull request. I will let you know if there are any angular issue that need to get done.

Yeah sure. Do let me know

**Comment by akshatmishra25** [comment]
> Sorry for the late delay @Biki-das @KIET7UKE and @akshatmishra25. it looks like @jobayer12 has already submitted a pull request. I will let you know if there are any angular issue that need to get done.

No issues. I would love to work on something similar in future. Btw thanks @Isaiah-Hamilton for taking out time to reply.
